src/main/scala/io/iohk/ethereum/crypto/ECDSASignature.scala

```diff
@@ -95,22 +39,72 @@ object ECDSASignature {
     ECDSASignature(r, s, v)
   }
 
-}
+  /**
+    * new formula for calculating point sign post EIP 155 adoption
+    * v = CHAIN_ID * 2 + 35 or v = CHAIN_ID * 2 + 36
+    */
```

Comment:
EIP 155 was adopted at some block. Is this wired up correctly with EvmConfig or that's another task?